requirements.txt

```diff
@@ -14,7 +14,7 @@ passlib==1.6.2
 python-dateutil==2.4.2
 python-mimeparse==0.1.4
 six==1.10.0
-git+git://github.com/xcompass/Flask-CAS.git
+git+git://github.com/cameronbwhite/Flask-CAS.git@v1.0.1
```

Comment:
Seems it now supports 2.0+ in [v1.0.0](https://github.com/cameronbwhite/Flask-CAS/releases/tag/v1.0.0). 

Also `CAS_LOGOUT_RETURN_URL` is available as `CAS_AFTER_LOGOUT`

I'll test it on verf to make sure it works